support/supportedversion/version.go

```diff
@@ -76,3 +80,39 @@ func IsValidReleaseVersion(version, currentVersion, latestVersionSupported, minS
 
 	return nil
 }
+
+type ocpVersion struct {
+	Name        string `json:"name"`
+	PullSpec    string `json:"pullSpec"`
+	DownloadURL string `json:"downloadURL"`
+}
+
+func LookupLatestSupportedRelease(ctx context.Context, releaseStream string) (string, error) {
+	prefix := fmt.Sprintf("https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/%s/latest", releaseStream)
```

Comment:
This needs to be a multi-arch release stream since we support Arm HCP and Arm NodePools.